index_search.go

```diff
@@ -65,11 +65,14 @@ func (i Index) Search(query string, request *SearchRequest) (*SearchResponse, er
 }
 
 func searchPostRequestParams(query string, request *SearchRequest) map[string]interface{} {
-	params := make(map[string]interface{}, 14)
+	params := make(map[string]interface{}, 16)
 
 	if !request.PlaceholderSearch {
 		params["q"] = query
 	}
+	if request.IndexUid != "" {
+		params["indexUid"] = request.IndexUid
+	}
```

Comment:
indexUid does not exist on simple search I think

Reply:
Yes, that why this condition is made for if there is any specified `IndexUid` it will not create the field

Reply:
Yes but is this not lying to the user? You tell them that `indexUid` is an accepted query parameter when it is not.

Reply:
Yes, it because I used the same struct `SearchResponse` than for the `Search()` and for `MultiSearch`